rules/windows/c2_mshta_beacon.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Mshta.exe making outbound network connections. This may indicate adversarial activity as Mshta is often
+leveraged by adversaries to execute malicious scripts and evade detection.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Mshta Making Network Connections"
```

Comment:
Idk if we want to get in the habit of adding `sequence` to all of the rules which use them, since it can generally be inferred by the eql `type`. I know I was the one to add it to the rules as a way to distinguish them from related kql rules, but maybe there is a better compromise in the middle of the two - so long as we are consistent.